boards/arm/nucleo_f401re/pinmux.c

```diff
@@ -41,6 +41,8 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PB14, STM32F4_PINMUX_FUNC_PB14_SPI2_MISO},
 	{STM32_PIN_PB15, STM32F4_PINMUX_FUNC_PB15_SPI2_MOSI},
 #endif	/* CONFIG_SPI_1 */
+#ifdef CONFIG_ADC_1
+#endif	/* CONFIG_ADC_1 */
```

Comment:
Is this empty `#ifdef` block intentional?

Reply:
Sorry, my mistake